crates/hc_sandbox/tests/cli.rs

```diff
@@ -167,25 +167,14 @@ async fn package_fixture_if_not_packaged() {
     println!("@@ Package Fixture Complete");
 }
 
-async fn clean_sandboxes() {
```

Comment:
This was used for cleaning the test folder at start of each test. Now with temp dirs this cleaning up is not needed anymore.